upstream-community-operators/kubeturbo/kubeturbo-operator.v6.3.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[{"apiVersion":"charts.helm.k8s.io/v1alpha1","kind":"Kubeturbo","metadata":{"name":"kubeturbo-example"},"spec":{"restAPIConfig":{"opsManagerPassword":"Turbo_password","opsManagerUserName":"Turbo_username"},"serverMeta":{"turboServer":"https://Turbo_server_URL","version":6.3}}}]'
+    capabilities: Basic Install
+  name: kubeturbo-operator.v6.3.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - kind: Kubeturbo
+      name: kubeturbos.charts.helm.k8s.io
+      version: v1alpha1
+  description: Placeholder description
+  displayName: Kubeturbo Operator
+  install:
+    spec:
+      deployments:
+      - name: kubeturbo-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: kubeturbo-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: kubeturbo-operator
+            spec:
+              containers:
+              - env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
```

Comment:
Please change this value to `metadata.annotations['olm.targetNamespaces']`

Reply:
glad to change this, can you please point me to another example like this to make sure I get this right?